storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/DbModule.java

```diff
@@ -39,16 +35,8 @@
     public StorIOSQLite provideStorIOSQLite(@NonNull SQLiteOpenHelper sqLiteOpenHelper) {
         return DefaultStorIOSQLite.builder()
                 .sqliteOpenHelper(sqLiteOpenHelper)
-                .addTypeMapping(Tweet.class, SQLiteTypeMapping.<Tweet>builder()
-                        .putResolver(new TweetStorIOSQLitePutResolver())
-                        .getResolver(new TweetStorIOSQLiteGetResolver())
-                        .deleteResolver(new TweetStorIOSQLiteDeleteResolver())
-                        .build())
-                .addTypeMapping(User.class, SQLiteTypeMapping.<User>builder()
-                        .putResolver(new UserStorIOSQLitePutResolver())
-                        .getResolver(new UserStorIOSQLiteGetResolver())
-                        .deleteResolver(new UserStorIOSQLiteDeleteResolver())
-                        .build())
+                .addTypeMapping(Tweet.class, new TweetSQLiteTypeMapping())
+                .addTypeMapping(User.class, new UserSQLiteTypeMapping())
```

Comment:
Can you please leave it as is to show that user may also specify resolvers independently? Probably worth adding a comment about it.